app/ui-react/syndesis/src/modules/data/pages/VirtualizationViewsPage.tsx

```diff
@@ -146,8 +166,8 @@ export const VirtualizationViewsPage: React.FunctionComponent = () => {
     await handlePublishVirtualization(pVirtualizationId, hasViews);
   };
 
-  const doUnpublish = async (virtualizationName: string) => {
-    await handleUnpublishVirtualization(virtualizationName);
+  const doUnpublish = async (serviceVdbName: string) => {
```

Comment:
this should be virtualizationName, not serviceVdbName

Reply:
Fixed